azurelinuxagent/ga/exthandlers.py

```diff
@@ -655,12 +625,13 @@ def wait_for_handler_completion(handler_i, wait_until, extension=None):
             msg = "Dependent Extension {0} did not succeed. Status was {1}".format(extension_name, status)
             raise Exception(msg)
 
-    def handle_ext_handler(self, ext_handler_i, extension, goal_state_id):
+    def handle_ext_handler(self, ext_handler_i, extension, goal_state_id, policy_engine):
```

Comment:
The policy agent should probably be a data member of ExtHandlersHandler instead of an argument to these functions